addons/event/views/event_event_views.xml

```diff
@@ -15,6 +15,15 @@
                 <header>
                     <button name="%(event_barcode_action_main_view)d"
```

Comment:
To avoid duplicating the button code, would pick either primary or secondary, probably the one that was there before your changes.

Reply:
Umm yes, I added the duplicate code because of changes requested by LNA :
![image](https://github.com/odoo/odoo/assets/124355728/46fa82bb-1efc-497b-a428-220b1c2bf918)

Undoing those changes would not serve his request. Do you want me to go ahead and keep the original code?